core/src/i18n/en.yaml

```
@@ -324,6 +327,8 @@ common:
     read-file: Read file content as a single value, with file name as a key.
     save: Save
     search: Search
+    secret-ref-name: Name is unique within a namespace to reference a secret resource.
```

```suggestion
    secret-ref-name: Unique name within a Namespace to refer to a Secret resource.
```
Is it Secret as in Kubernet Secret (then we need to capitalize), or another kind of secret?

---

Yes it is